instantiate docs and initial attempt at glossary (#2)
statistical-perceptions/survey-builder: docs/source/glossary.rst

Glossary
--------

Key terms used throughout the project

.. glossary::
  :sorted:

  study

>     a fully configured experiment for a participant that consists of FIXME

Author: ```suggestion
    a fully configured experiment for a participant that consists of qualtrics surveys and an experiment on this platform including a specific cohort of subjects
```

  instance
    a deployment of the statistical perceptions platform consisting contained in
    one Heroku app. An instance of the app consists of one set of databases.

  experiment

>     a template for studies. a given experiment can be run many times with
>     different settings for different studies. an experiment consists of a set
>     of items of a given type but may have different values of parameters

Author: ```suggestion
    a group of items with all parameters set. 
```

  app

>     FIXME as in app1 and app2

Reviewer: a website, on which experiments can be either designed and saved, or on which experiments can be run by survey respondents

>     FIXME as in app1 and app2

Author: ```suggestion
    a website, on which experiments can be either designed and saved, or on which experiments can be run by survey respondents
```
